docs/pywbemcli/commands.rst

```diff
@@ -2999,6 +3044,12 @@ the current WBEM server.
 class so that any indication destination instances defined with the superclass
 ``CIM_ListenerDestination`` are not visible.
 
+The format of this command is:
+
+.. code-block:: text
+
+    pywbemcli [GENERAL-OPTIONS] subscription list-destinations [COMMAND-OPTIONS]
```

Comment:
For commands in the shell, show a `$` prompt.

Reply:
Fixed those.